util/include/memory.hpp

```diff
@@ -12,6 +12,7 @@
 //
 #pragma once
 #include <memory>
+#include <openssl/crypto.h>
```

Comment:
remove this include

Reply:
Removed in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2